src/main/java/org/greenbuttonalliance/gbaresourceserver/usage/model/IntervalBlock.java

```diff
@@ -45,7 +45,7 @@ public class IntervalBlock extends IdentifiedObject {
 	@Embedded
 	private DateTimeInterval interval;
 
-	@OneToMany(mappedBy = "block", cascade = CascadeType.ALL)
+	@OneToMany(mappedBy = "block", cascade = CascadeType.ALL, orphanRemoval = true)
```

Comment:
This addition means that, if (in this particular case), an `IntervalBlock` with 3 `IntervalReading`s is retrieved from the database, then one of the `IntervalReading`s is removed from the list and the `IntervalBlock` is saved back to the database, the `IntervalReading` removed from the list will be dropped from the database as well. Again, probably not a necessary change, but might be useful in the future.